### ISSUE TITLE
Add MSI downloads to FoxitReader

### ISSUE THREAD

**Issue by DanGough** [aaronparker/Evergreen#143]
FoxitReader currently returns a bunch of language-specific exe files that are small web stubs that cannot be installed offline.

They offer MSI downloads - you do have to provide your details to get the links, but they work in incognito mode so you don't need to actually login to download. For example:

https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_enu_Setup.msi

**Comment by aaronparker** [comment]
Updated in development:

```
Version      Date      Language   URI
-------      ----      --------   ---
10.1.3.37598 22/3/2021 Danish     https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
10.1.3.37598 22/3/2021 Dutch      https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
10.1.3.37598 22/3/2021 English    https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_enu_Setup.msi
10.1.3.37598 22/3/2021 Finnish    https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
10.1.3.37598 22/3/2021 French     https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
10.1.3.37598 22/3/2021 German     https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
10.1.3.37598 22/3/2021 Italian    https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
10.1.3.37598 22/3/2021 Korean     https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
10.1.3.37598 22/3/2021 Norwegian  https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
10.1.3.37598 22/3/2021 Polish     https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
10.1.3.37598 22/3/2021 Portuguese https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
10.1.3.37598 22/3/2021 Russian    https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
10.1.3.37598 22/3/2021 Spanish    https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
10.1.3.37598 22/3/2021 Swedish    https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
10.1.3.37598 22/3/2021 Zhtw       https://cdn01.foxitsoftware.com/product/reader/desktop/win/10.1.3/FoxitReader1013_L10N_Setup.msi
```

**Comment by aaronparker** [comment]
Released in `2104.355`
